src/components/menus/content-contextual-menu.js

```diff
@@ -496,10 +497,12 @@ const ContentContextualMenu = (props) => {
 
     const allowsDownloadCase = useCallback(() => {
         //if selectedElements contains at least one case
-        return selectedElements.some(
-            (element) => element.type === ElementType.CASE
+        return (
+            selectedElements.some(
+                (element) => element.type === ElementType.CASE
+            ) && noCreationInProgress()
         );
-    }, [selectedElements]);
+    }, [selectedElements, noCreationInProgress]);
 
     const handleDownloadCases = useCallback(async () => {
         const casesUuids = selectedElements
```

Comment:
instead of adding the condition in each allow[...] functions, isn't it better to add it as a parent of all the following ```if (allowsRename()){```, etc to avoid forgetting to handle this in the next addition as what happened with the Download?

Reply:
well, I dont see what we can do properly. Almost each allowXY() function is specific, so I dont see what I can put in common somewhere above.